SCClassLibrary/JITLib/ProxySpace/NodeProxy.sc

```diff
@@ -3,7 +3,7 @@ NodeProxy : BusPlug {
 	var <group, <objects, <nodeMap, <children;
 	var <loaded=false, <>awake=true, <paused=false;
 	var <>clock, <>quant;
-	classvar <>buildProxyControl, <>buildProxy;
+	classvar <>buildProxyControl, <>buildProxy, <defaultFadeTime=0.02;
```

Comment:
just noticed this has a public accessor -- should have documentation or be hidden

Reply:
ok, done!